openai-core/src/commonMain/kotlin/com.aallam.openai.api/image/ImageCreation.kt

```diff
@@ -11,6 +12,10 @@ public class ImageCreation(
      * A text description of the desired image(s). The maximum length is 1000 characters.
      */
     public val prompt: String,
+    /**
+     * The model used to generate image. Must be one of dall-e-2 or dall-e-3. If not provided, dall-e-2 is used.
+     */
+    public val model: ModelId? = null,
```

Comment:
I just remembered something: new fields should always be added at the end, to not break compatibility with users not using named parameters!

Reply:
Sure thing, it makes sense.